### PR TITLE
Kg/k module

### PR DESCRIPTION
**Pull request by kagarmoe**

### CODE REVIEW

File: docs/resources/kernel_module.md.erb [hunk 1]
```diff
@@ -39,6 +40,12 @@ where
 
 The following examples show how to use this InSpec audit resource.
 
+### version
+
+The `version` property tests if the named module version is on the system:
+
+    its(:version) { should eq '3.2.2' }
```
Comment (jquick) [inline docs/resources/kernel_module.md.erb]: Do we have other resources with symbol properties? I confirmed 'version' works as well.

File: docs/resources/kernel_module.md.erb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: About the kernel_module Resource
 platform: linux
+=======
```
Comment (jerryaldrichiii) [inline docs/resources/kernel_module.md.erb]: Is this line needed? Could we remove it?

File: docs/resources/kernel_module.md.erb [hunk 3]
```diff
@@ -40,11 +41,17 @@ where
 
 The following examples show how to use this InSpec audit resource.
 
+### version
+
+The `version` property tests if the named module version is on the system:
+
+    its('version') { should eq '3.2.2' }
+
 ### Test a modules 'version'
```
Comment (jerryaldrichiii) [inline docs/resources/kernel_module.md.erb]: Not part of your change but could we change this to `module's`

File: docs/resources/kernel_module.md.erb [hunk 4]
```diff
@@ -40,11 +41,17 @@ where
 
 The following examples show how to use this InSpec audit resource.
 
+### version
+
+The `version` property tests if the named module version is on the system:
```
Comment (jerryaldrichiii) [inline docs/resources/kernel_module.md.erb]: I would reword this to something like:

```
The `version` property tests if the named module on the system has the correct version:
```

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]
Looks great @kagarmoe just a few suggestions/questions.

File: docs/resources/kernel_module.md.erb [hunk 5]
```diff
@@ -94,14 +100,21 @@ The following examples show how to use this InSpec audit resource.
...
+### be_disabled
 
-    its(:version) { should eq '3.2.2' }
+The `be_disabled` matcher tests if the module is disabled:
+
+    it { should be_disabled }
+
+### be_loaded
+
+The `be_loaded` matcher tests if the module is a loadable kernel module:
```
Comment (jerryaldrichiii) [inline docs/resources/kernel_module.md.erb]: Could we change this to:

```
The `be_loaded` matcher tests if the named module is loaded:
```

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]
Great job! Only one suggestion.

**Review by jerryaldrichiii — APPROVED** [review]
Looks great @kagarmoe thank you for all the iterations.

**Review by jquick — APPROVED** [review]
Thanks @kagarmoe !
